examples/gallery/symbols/bars.py

```diff
@@ -0,0 +1,68 @@
+"""
+Vertical or horizontal bar
+--------------------------
+The :meth:`pygmt.Figure.plot` method can plot vertical (**b**) or
+horizontal (**B**) bars by passing the corresponding shortcut to
+the ``style`` parameter. To plot multi-band bars, please append
+**+v**|**i** *ny* (for verticals bars) or **+v**|**i** *nx*
+(for horizontal ones), where *ny* or *nx* indicate the total
+number of bands in the bar. Here, **+i** means we must accumulate
+the bar values from the increments *dy* or *dx*, while *+v* means
```

Comment:
I'm a little confused about *dy* and *dx*.

Reply:
Meaning you suggest a more detailed explanation?

Reply:
*dx* and *dy* only appear one time in the tutorial, so it's unclear to me what *dy* and *dx* mean and how to specify *dx* and *dy*. I also read the GMT documentation and still don't understand them.

Reply:
In fact, the explanation/description what *nx* and *ny* are and how they are specified is not very clear @seisman.  Maybe @PaulWessel can help out in explaining it in a more simple way?

Reply:
_nx_ or _ny_ are the number of z-values expected in the input to build the stacked bar.  So as the _bar2dmultimode.sh_ example shows, **+i**4 means there needs to be 4 values of increments following the (x,y) coordinates.  Perhaps it would help if the docs said 

...where _nx_ or _ny_ indicate the total number of bands in the bar (and hence the number of values required to follow  the _x,y_ coordinate pair in the input).

Let me know and I can drop that in.

Reply:
Thanks @PaulWessel.

Reply:
I added that phrase and pushed directly to master.

Reply:
Ping @seisman for another round of review.